src/entities/UserSession.ts

```diff
@@ -0,0 +1,83 @@
+import crypto from 'crypto';
+import {
+  Column,
+  Entity,
+  JoinColumn,
+  OneToOne,
+  PrimaryGeneratedColumn,
+} from 'typeorm';
+import { v4 as uuidv4 } from 'uuid';
+import { SerializedUserSession } from '../common/types';
+import User from './User';
+
+@Entity('usersessions')
+class UserSession {
+  /** Universally unique identifier */
+  @PrimaryGeneratedColumn('uuid')
+  uuid: string = uuidv4();
+
+  /** Access token associated with session */
+  @Column('character varying')
+  accessToken = '';
```

Comment:
newer versions of typeorm (like the one you are using) recommend setting column defaults using column parameters instead of initializing the value

```
@Column({ type: 'character varying', default: '' })
```